var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -110,6 +120,12 @@ class Mpich(AutotoolsPackage):
     depends_on("m4", when="@develop", type=("build")),
     depends_on("autoconf@2.67:", when='@develop', type=("build"))
 
+    # building with "+hwloc' also requires regenerating autotools files
+    depends_on('automake@1.15:', when='+hwloc', type=("build"))
+    depends_on('libtool@2.4.4:', when='+hwloc', type=("build"))
+    depends_on("m4", when="+hwloc", type=("build")),
+    depends_on("autoconf@2.67:", when='+hwloc', type=("build"))
```

Comment:
```suggestion
    depends_on('automake@1.15:', when='+hwloc', type="build")
    depends_on('libtool@2.4.4:', when='+hwloc', type="build")
    depends_on("m4", when="+hwloc", type="build"),
    depends_on("autoconf@2.67:", when='+hwloc', type="build")
```
`(foo)` is actually the same as `foo` (which is fine in this case). You need to use `(foo,)` to make it a proper tuple.